app/bundles/ApiBundle/Event/ApiSerializationContextEvent.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace Mautic\ApiBundle\Event;
+
+use FOS\RestBundle\Context\Context;
+use Mautic\CoreBundle\Event\CommonEvent;
+use Symfony\Component\HttpFoundation\Request;
+
+class ApiSerializationContextEvent extends CommonEvent
```

Comment:
```suggestion
final class ApiSerializationContextEvent extends CommonEvent
```
There is no need for a DTO object to be extensible.